aws/resource_aws_codebuild_project.go

```diff
@@ -23,6 +23,9 @@ func resourceAwsCodeBuildProject() *schema.Resource {
 		Read:   resourceAwsCodeBuildProjectRead,
 		Update: resourceAwsCodeBuildProjectUpdate,
 		Delete: resourceAwsCodeBuildProjectDelete,
+		Importer: &schema.ResourceImporter{
```

Comment:
Can you please document the new import support at the bottom of `website/docs/r/codebuild_project.html.markdown`? Feel free to copy-paste an existing `## Import` section from another resource and adjust appropriately.